pkg/server/server.go

```diff
@@ -45,6 +46,19 @@ func getAppenders(currMachineConfig string, f kubeconfigFunc, osimageurl string)
 	return appenders
 }
 
+// machineConfigToIgnition converts a MachineConfig object into Ignition.
+func machineConfigToIgnition(mccfg *mcfgv1.MachineConfig) *igntypes.Config {
+	tmpcfg := mccfg.DeepCopy()
+	spec := &tmpcfg.Spec.Config
+	tmpcfg.Spec.Config = igntypes.Config{}
```

Comment:
Could we use NewIgnConfig() instead (from controller/common/helpers.go) for consistency and gen a valid ign confg base?

Reply:
Sure, done